presto-spark-base/src/test/java/com/facebook/presto/spark/task/UnsupportedOutputBuffer.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.task;
+
+import com.facebook.presto.execution.Lifespan;
+import com.facebook.presto.execution.StateMachine;
+import com.facebook.presto.execution.buffer.BufferResult;
+import com.facebook.presto.execution.buffer.BufferState;
+import com.facebook.presto.execution.buffer.OutputBuffer;
+import com.facebook.presto.execution.buffer.OutputBufferInfo;
+import com.facebook.presto.execution.buffer.OutputBuffers;
+import com.facebook.presto.spi.page.SerializedPage;
+import com.google.common.util.concurrent.ListenableFuture;
+import io.airlift.units.DataSize;
+
+import java.util.List;
+import java.util.function.Consumer;
+
+class UnsupportedOutputBuffer
```

Comment:
This looks very similar to com.facebook.presto.sql.planner.TestLocalExecutionPlanner.TestingOutputBuffer

Can we reuse the code?